test/processor.test.ts

```diff
@@ -19,7 +19,7 @@ const handle = (event: FirehoseTransformationEvent) =>
 const buildFirehoseEvent = (records: FirehoseTransformationEventRecord[]) =>
   firehoseEventFactory.build({ records });
 
-const buildKinesisRecord = (id: string, ocsEventAttrs: Partial<OCSEvent>) =>
+const buildKinesisRecord = (id: string, ocsEventAttrs: any) =>
```

Comment:
question: is `any` the best type here, or could we specify an object type that has at least the expected properties?

Reply:
> question: is `any` the best type here, or could we specify an object type that has at least the expected properties?

I will poke this because my knowledge of TypeScript is somewhat limited but my understanding is either we would have to create another object that is only used in testing that allows arbitrary fields in addition to some hardcoded fields, which would functionally not test anything - or we use the any keyword and allow the record to pass through this function with whatever we want it to have via `any`. 

This may be pessimistic so I am going to poke it like I said 😄

Reply:
@bklebe Just in case - did you see this is only being applied in a test suite to allow for mock data from Kinesis? 

This needs to be relatively loose so I can pass through the `sourceip` and have it get correctly discarded without an error by the `mask` function in the actual processing loop. 

I changed it up to be `Record<PropertyKey, string | number | object>` instead of just `any`, which is more reflective of what we want (a Kinesis payload could technically have any of those things.